plugins/CoreHome/vue/src/Sparkline/Sparkline.vue

```diff
@@ -19,7 +19,9 @@ import { format } from '../Periods';
 export default defineComponent({
   props: {
     seriesIndices: Array,
-    params: Object,
+    params: [Object, String],
+    width: Number,
+    height: Number,
```

Comment:
The sparkline on the reporting page is displayed too big.
The twig template used `width=200`, `height=50`. Not sure if we should use those as default values here, or pass them through for each sparkline.